Cargo.toml

```diff
@@ -13,11 +13,13 @@ keywords = ["encryption", "xts"]
 exclude = ["test_files"]
 
 [features]
+default = ["std"]
+std = ["byteorder/std", "cipher/std"]
```

Comment:
As we don't currently use their std features, can we just remove both from here?

```
std = []
```